samples/MiniZinc/data.dzn

```diff
@@ -0,0 +1 @@
+n=3;
```

Comment:
> and copied the MIT license appropriate to those samples to their folders.

Linguist won't be able to handle a non MiniZinc file in the `MiniZinc` directory :/

@brandonblack Until now, we haven't included licenses of sample files, even though most of them require it. Do you know if that's okay? (Maybe we should ask talniv or mlinksva?)